### PR TITLE
Run tests in mini-python

### PR DESCRIPTION
**Pull request by jhbaarnh**
This is required by https://github.com/intel-innersource/applications.simulators.simics.simics-base/pull/2805

### CODE REVIEW

**Comment by syssimics** [comment]
Verification [#10783556](https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=10783556): fail

File: test/1.2/events/T_dup.py [hunk 1]
```diff
@@ -4,15 +4,23 @@
...
 obj.queue = cpu
 obj.postall = 1
 
 SIM_write_configuration_to_file("dup.chkp", Sim_Save_Nobundle)
 
+if os.path.isdir(sys.argv[0]):
+    exe_sfx = '.exe' if is_windows() else ""
+    app = [join(sys.argv[0], '..', 'mini-python') + exe_sfx, sys.argv[0]]
+else:
+    app = [sys.argv[0]]
```
Comment (mandolaerik) [inline test/1.2/events/T_dup.py:19]: maybe even easier to drop argv and just run `f'{conf.sim.project}/bin/simics'`?

Comment (jhbaarnh) [reply test/1.2/events/T_dup.py]: Fixed

File: test/1.4/events/T_after_chk.py [hunk 2]
```diff
@@ -4,15 +4,23 @@
...
+if os.path.isdir(sys.argv[0]):
+    exe_sfx = '.exe' if is_windows() else ""
+    app = [join(sys.argv[0], '..', '..', 'mini-python') + exe_sfx, sys.argv[0]]
+else:
+    app = [sys.argv[0]]
+
 subprocess.check_call(
-    [sys.argv[0], "-batch-mode", "-quiet", "-no-copyright", "-core", "-werror",
+    app +
+    ["-batch-mode", "-quiet", "-no-copyright", "-core", "-werror",
```
Comment (mandolaerik) [inline test/1.4/events/T_after_chk.py:23]: Same issue here

File: test/1.2/events/T_after_chk_dup.py [hunk 3]
```diff
@@ -4,15 +4,23 @@
...
+if os.path.isdir(sys.argv[0]):
+    exe_sfx = '.exe' if is_windows() else ""
+    app = [join(sys.argv[0], '..', '..', 'mini-python') + exe_sfx, sys.argv[0]]
+else:
+    app = [sys.argv[0]]
+
 subprocess.check_call(
-    [sys.argv[0], "-batch-mode", "-quiet", "-no-copyright", "-core", "-werror",
+    app +
+    ["-batch-mode", "-quiet", "-no-copyright", "-core", "-werror",
```
Comment (mandolaerik) [inline test/1.2/events/T_after_chk_dup.py:23]: And here

File: test/1.4/events/T_after_chk.py (hunk 2 above)
Comment (jhbaarnh) [reply test/1.4/events/T_after_chk.py]: Done

File: test/1.2/events/T_after_chk_dup.py (hunk 3 above)
Comment (jhbaarnh) [reply test/1.2/events/T_after_chk_dup.py]: Done

**Review by mandolaerik — APPROVED** [review]

**Comment by syssimics** [comment]
Verification [#11008574](https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=11008574): pass

**Pull request merged**